apps/nowcasting-app/components/helpers/utils.ts

```diff
@@ -173,15 +176,30 @@ export const getRounded4HoursAgoString = () => {
   return convertISODateStringToLondonTime(fourHoursAgo.toISOString());
 };
 
+export const getRoundedPv = (pv: number, round: boolean = true) => {
+  if (!round) return Math.round(pv);
+  // round To: 0, 100, 200, 300, 400, 500
+  return Math.round(pv / 100) * 100;
+};
+export const getRoundedPvPercent = (per: number, round: boolean = true) => {
```

Comment:
hi @braddf, you might have already fixed this, but I think the map on the `%` being all yellow might be related to this function. 

I changed `/10` in lines 189 - 191 to `/100` and that seems to be better, but I'm not sure on the maths if that's corrected the problem or created a new one.

Reply:
otherwise this is looking good and seems to be loading a lot faster!

Reply:
if we `/100` does it then round to 100's not 10's?

Reply:
yea, could the problem be a bit deeper? ie. not in this bit of code

Reply:
Thanks for the comments and digging both, sounds like something reasonable-yet-awry is going on here... just fixing/checking and refactoring some stuff on the aggregation branch ready for merging and then I'll take a look at this. Screaming out for some unit tests methinks! 😂

Reply:
@braddf, I'll do a bit more digging. but we could look together / pair code tomorrow if you have time?